pallets/ds-maps/src/tests.rs

```diff
@@ -0,0 +1,90 @@
+use crate::mock::*;
+
+use frame_support::{
+    assert_noop, assert_ok,
+};
+
+type Error = super::Error<Test>;
+
+// Constants to make tests more readable
+const ADMIN_ACCOUNT_ID: u64 = 1;
+const REGISTRAR_1_ACCOUNT_ID: u64 = 2;
+//if we will stick to the global coords =>
+//55.123532 will be represented as 55123532 
```

Comment:
These things must be described in some appropriate doc comment, presumably in `Point3D`'s docs, not just in a test module.